DSCClassResources/Printer/Printer.psm1

```diff
@@ -57,6 +57,18 @@ class Printer
     [System.String]
     $lprQueueName
 
+    [DscProperty()]
+    [System.String]
+    $Location
+
+    [DscProperty()]
+    [System.String]
+    $Comment
+
+    [DscProperty()]
+    [System.Boolean]
+    $Published = $false
```

Comment:
Can you update the readme to include the Published property

Reply:
sure thing